k8s/Dockerfile-azure

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04 as base
+k8s/Dockerfile-azure FROM ubuntu:18.04 as base
```

Comment:
I'm actally not sure how this sneakedi n.

Reply:
lol, we're removing this file anyway.